main.js

```diff
@@ -1,81 +1,78 @@
-const bookList = document.querySelector('.bookList');
+const listBooks = document.querySelector('.bookList');
 const form = document.querySelector('.formInput');
 const [title, author] = form.elements;
 
-const bookInput = {}; // empty object
-let books = []; // empty array
+const inputBook = {};
+let books = new Array([]);
```

Comment:
- [ ] Great job working on refactoring the code! 💯 However, this line is causing an unexpected "undefined" book to be created. When you instantiate an Array with `new Array()` you don't actually have to pass the brackets `[]`, doing so will result in the following: `[ [] ]`, the outer array being the one created by the constructor `new` and the element inside being the argument you passed to that constructor (inside the parenthesis), in this case, an empty array, so please consider removing it! 🤓 🔧 

```suggestion
let books = new Array();
```